shopinvader/services/service.py

```diff
@@ -120,3 +122,18 @@ def _get_openapi_default_parameters(self):
             }
         )
         return defaults
+
+    @property
+    def shopinvader_session_data(self):
+        """
+        A mutable dictionary which is injected in service responses.
+        """
+        return session.get()
```

Comment:
Actually (and as this method docstring says), this is not specifically a session. It's just stuff that we can inject in the response from anywhere in the code processing the request. It is the content we put into it that is specific to shopinvader (`set_session`, `store_cache`, ...). So perhaps `session` should be renamed and it, together with `dispatch` override, could go in an independent module.

The it is how the shopinvader uses it that gives it a specific session meaning, and we should have higher level functions in shopinvader such as set_session, set_store_cache, ...